### PR TITLE
Add supported k8s versions to istio status

### PR DESCRIPTION
**Pull request by jacob-delgado**
List the versions of k8s supported by Istio releases.

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/about/supported-releases/index.md [hunk 1]
```diff
@@ -11,12 +11,12 @@ changes. For more information refer to the [Istio support policy](../release-cad
...
-| Version         | Currently Supported   | Release Date    | End of Life       |
-|-----------------|-----------------------|-----------------|-------------------|
-| master          | No, development only  |                 |                   |
-| 1.7             | Yes                   | August 21, 2020 |                   |
-| 1.6             | Yes                   | May 21, 2020    | November 21, 2020 |
-| 1.5 and earlier | No                    |                 |                   |
+| Version         | Currently Supported   | Release Date    | End of Life       | Supported Kubernetes Versions |
+|-----------------|-----------------------|-----------------|-------------------|-------------------------------|
+| master          | No, development only  |                 |                   |                               |
+| 1.7             | Yes                   | August 21, 2020 |                   | 1.15, 1.16, 1.17, 1.18, 1.19  |
```
Comment (linsun) [inline content/en/about/supported-releases/index.md:17]: I think 1.15 is not supported.  Also, neither is 1.19.  Could you double check our docs?

Comment (jacob-delgado) [reply content/en/about/supported-releases/index.md]: Double checked https://istio.io/latest/faq/general/#what-deployment-environment and added 1.8 info

**Comment by jacob-delgado** [comment]
> Agreed, would be nice to publish the future one, at least 1 release ahead when that data is avail

Added 1.8 since it was TOC approved today

**Review by linsun — APPROVED** [review]
LGTM

Thank you @jacob-delgado

**Review by ericvn — APPROVED** [review]
I know this is what we stated in the design doc, but did we say we supported 1.15 as well in a TOC meeting for 1.7? I couldn't find it doc'd anywhere.

**Comment by jacob-delgado** [comment]
> I know this is what we stated in the design doc, but did we say we supported 1.15 as well in a TOC meeting for 1.7? I couldn't find it doc'd anywhere.

I thought we said this was going to be unofficially supported. I know there's a major contributor that runs 1.15 in prod and they are making sure it works off of master with k8s 1.15. But I think that was more of a "doesn't break" on 1.15 and isn't relegated to officially supported.

**Pull request merged**